index.html

```diff
@@ -1495,12 +1495,12 @@ <h3>
           <var>manifest</var>["<a>categories</a>"].
           </li>
           <li>Set <var>manifest</var>["<a>icons</a>"] to the result of running
-          <a>processing ImageResource members</a> given
+          <a>processing <code>ImageResource</code> members</a> given
```

Comment:
I believe we can use backtick instead of <code> in respec today

Reply:
I’ll test. I wasn’t sure if markdown was supported now.

Reply:
It is.

Reply:
Check the wake lock spec for use of modern Respec features :-)